Libraries/Components/View/ViewPropTypes.windows.js

```diff
@@ -497,4 +497,22 @@ module.exports = {
         'scale',
         'all',
     ])),
+
+  /**
+   * Enables the view as a drop target. If this is set, the user will be able
+   * to drag and drop files to the view and it will emit the four events:
+   *
+   *    `onDragEnter`
+   *    `onDragOver`
+   *    `onDrop` - contains the list of dropped files (their URIs to be precise)
+   *    `onDragLeave`
+   *
+   * The native event in these callbacks contains the `dataTransfer` field that
+   * implements the DataTransfer JS interface. Note, that unlike in the web, here
+   * the event is readonly and thus changing `effectAllowed` and `dropEffect` fields
+   * will do nothing.
+   *
+   * @platform windows
+   */
+  allowDrop: PropTypes.bool,
```

Comment:
Put this in `PlatformViewPropTypes.windows.js`

Reply:
Fixed